### PR TITLE
Add Protobuf example

### PR DESCRIPTION
**Pull request by uilianries**
- Copy https://github.com/uilianries/conan-protobuf-example to protobuf
- Update README with example list

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+## Several conan examples, to extend the docs.
+
+
+# Examples
+
+#### [Workspace](workspace)
+
+Example how to use Conan Workspaces.
+
+Documentation: https://docs.conan.io/en/latest/developing_packages/workspaces.html
```
Comment (danimtb) [inline README.md]: Cool! I like the links 😄

File: protobuf/.gitignore [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Prerequisites
```
Comment (danimtb) [inline protobuf/.gitignore]: gitignore should be placed in the root of the repo?

File: protobuf/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+os: linux
```
Comment (danimtb) [inline protobuf/.travis.yml]: Travis needed? 😕

File: protobuf/LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (danimtb) [inline protobuf/LICENSE]: Repo already has a license

File: protobuf/.travis.yml (hunk 3 above)
Comment (uilianries) [reply protobuf/.travis.yml]: No, just to check if I can compile the project. I gonna remove it to make the example simple as possible.

File: protobuf/LICENSE (hunk 4 above)
Comment (uilianries) [reply protobuf/LICENSE]: removed

File: protobuf/.gitignore (hunk 2 above)
Comment (uilianries) [reply protobuf/.gitignore]: should we add a git ignore file? maybe just removing it is enough.

Comment (danimtb) [reply protobuf/.gitignore]: Maybe it could make sense to have a reduced one, but let's clear this file and do that in the future if needed. Thanks!

**Pull request merged**